workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/hooks/useRecentTickets.ts

```diff
@@ -24,11 +24,16 @@ const useRecentTickets = () => {
 				}
 				catch (error) {}
 
+				delete (ticket as any).actions;
```

Comment:
Don't you have a ticket type you can use here instead of `any`?

Reply:
Or in fact... this already has the ticketPayload type... I'm guessing that means that TicketPayload, probably just needs to have actions as an optional property.